src/resources/views/media/_index.blade.php

```diff
@@ -1,18 +1,17 @@
 <?php $media = $model->getMedia($collection ?? 'default') ?>
-<div class="card">
-    <div class="card-body">
 
-        <h6 class="card-title">
-            {{ __('Images') }}
-            <span class="badge badge-pill badge-dark">{{ $media->count() }}</span>
-        </h6>
+<x-appshell::card accent="secondary">
```

Comment:
This card had no accent previously, so it should be without an accent:
```suggestion
<x-appshell::card>
```